recipes/nrtk-explorer/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "nrtk-explorer" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nrtk_explorer-{{ version }}.tar.gz
+  sha256: 7e7c915da91a3b5d2124889f435ddf121dd35079c364e8c8571031490e724686
+
+build:
+  entry_points:
+    - nrtk-explorer = nrtk_explorer.app:main
+    - nrtk-explorer-embeddings = nrtk_explorer.app.embeddings:embeddings
+    - nrtk-explorer-tranforms = nrtk_explorer.app.transforms:transforms
+    - nrtk-explorer-filtering = nrtk_explorer.app.filtering:filtering
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - hatchling
+    - nodejs
+    - pip
+    - python >=3.7
+  run:
+    - accelerate
+    - nrtk
+    - numpy
+    - pillow
+    - python >=3.7
+    - pytorch
+    - scikit-image
+    - scikit-learn ==1.4.2
+    - smqtk-classifier ==0.19.0
+    - smqtk-core ==0.19.0
+    - smqtk-dataprovider ==0.18.0
+    - smqtk-descriptors ==0.19.0
+    - smqtk-detection ==0.20.1
+    - smqtk-image-io ==0.17.1
+    - tabulate
+    - timm
+    - torchvision
+    - trame
+    - trame-client >=2.15.0
+    - trame-quasar
+    - trame-server >=2.15.0
+    - transformers
+    - ubelt ==1.3.5
+    - umap-learn
+    - __unix
```

Comment:
If the only reason for this is a missing dependency, this can be removed.

There is some more info here: https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python